src/main/java/duke/command/AddCommand.java

```diff
@@ -0,0 +1,78 @@
+package duke.command;
+
+import java.time.LocalDate;
+
+import duke.storage.Storage;
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.TaskList;
+import duke.task.Todo;
+
+/**
+ * This AddCommand class represents a command to add a task to the task list.
+ */
+public class AddCommand extends Command {
```

Comment:
AddCommand sounds like an action. Will CommandAdder sound more like a class? Same goes for other subclasses of Command.